src/pages/stablecoins.tsx

```diff
@@ -526,7 +526,7 @@ const StablecoinsPage = ({ markets, marketsHasError }) => {
                   </Text>
                   <Flex direction="column">
                     <Box>
-                      <ButtonLink mb={4} me={4} to="https://1inch.exchange">
+                      <ButtonLink mb={4} me={4} target="_blank" href="https://matcha.xyz/tokens/ethereum/0x6b175474e89094c44da98b954eedeac495271d0f?sellChain=1&sellAddress=0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee">
```

Comment:
No need for this, our link component automatically adds `target="_blank"` to external links :) 

```suggestion
                      <ButtonLink mb={4} me={4} href="https://matcha.xyz/tokens/ethereum/0x6b175474e89094c44da98b954eedeac495271d0f?sellChain=1&sellAddress=0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee">
```

Reply:
Ah yes! My apologies, removed both in the latest commit. @minimalsm